### PR TITLE
Add initial governance guidelines

### PR DESCRIPTION
**Pull request by achantavy**
Per https://github.com/cncf/sandbox/issues/135, including governance guidelines.

### CODE REVIEW

**Comment by ramonpetgrave64** [comment]
Looks good, but how do we register emeritus contributors.

**Comment by achantavy** [comment]
> Looks good, but how do we register emeritus contributors.

By adding them to here: https://github.com/cartography-cncf/cartography/blob/master/MAINTAINERS.md#emeritus-maintainers

Think I should make that explicit in the doc or nah?

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- Contributing 2+ PRs to Cartography.
+- Regularly participating in discussion on GitHub, on Slack, or in our live video meetings.
+
+# Maintainer responsibilities
+- Hang out on Slack; delayed response is perfectly acceptable
+- Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+- Deeply understand the project's best practices and development patterns.
+
+# When does a maintainer lose maintainer status
+- If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status.
```
Comment (ramonpetgrave64) [inline GOVERNANCE.md:12]:
```suggestion
- If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to the list of emeritus maintainers.
```

**Review by ramonpetgrave64 — APPROVED** [review]

**Comment by ramonpetgrave64** [comment]
@achantavy I think so, or add a link.

**Pull request merged**